docs/goodtoknow.rst

```diff
@@ -48,15 +48,19 @@ What is currently not covered by either `xscen` or `xclim` is a method to resamp
 Metadata translation
 --------------------
 
-xscen itself does not add many translatable attributes, but when it does, it will look into xclim's options for which locales to translate them to. In a config file, activating French translations for both xclim's indicators and xscen is done with :
+xscen itself does not add many translatable attributes, but when it does, it will look into xclim's options for which locales to translate them to. Similar to xclim, it will always add a particular attribute in english and then translations with the same attribute name followed by "_XX" where "XX" is the two-letter locale name, usually following te `ISO-639-1 <https://en.wikipedia.org/wiki/ISO_639-1>`_ standard. For example, if a function adds a `long_name` and Inuktitut translation is activated, the function will also add a `long_name_iu` attribute.
```

Comment:
typo following tHe

(sorry it is not letting me do a suggestion)

Reply:
Also, we shouldn't _usually_ follow ISO-639-1, we should strictly follow it.

[am_I_the_only_one_who_cares_about_the_rules.png]

Reply:
Haha, well. When we made this decision in xclim, we went with our first intuition : "_fr", the ISO-639-1 code. But, the current CF-conventions proposal (244 on that github) suggests ISO-639-2, the three letter version. The latter has a lot more languages than the former.

I'll change my sentence, but I wanted to raise this "warning" that we might have to change our standard if the CF-conventions decide to follow ISO-639-2.

Reply:
Good to know!

Reply:
Oh and I just realize that python's `gettext` and `babel` currently work with [RFC 3066](https://datatracker.ietf.org/doc/html/rfc3066.html) which uses ISO-639-1 as the language tag. We'll see what CF decides!